packages/bot-skeleton/globals.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module '@deriv/account';
+declare module '@deriv/bot-web-ui';
+declare module '@deriv/cashier';
+declare module '@deriv/components';
+declare module '@deriv/appstore';
+declare module '@deriv/p2p';
+declare module '@deriv/trader';
+declare module '@deriv/translations';
+declare module '@deriv/shared';
```

Comment:
I think we only use '@deriv/shared', '@deriv/translations', '@deriv/components' in bot packages. Can you remove the ones that we are not using.

Reply:
Could you please check the modules again I think we only use these:
'@deriv/shared'
'@deriv/translations'
'@deriv/indicators'